fairship.sh

```diff
@@ -13,6 +13,7 @@ requires:
   - ROOT
 build_requires:
   - googletest
+  - madgraph5
```

Comment:
This should go under `requires`, not `build_requires`

Reply:
Why? If I understand aliBuild correctly, if it is under build_requires it does not included into build dependences for FairShiP, which is true, because you don't want to rebuild you FairShip every time you changed MadGraph. Or, my understanding is wrong?

Reply:
`build_requires` are needed at build time, `requires` at runtime. I think either will trigger a rebuild (to be checked?). In any case, rebuilding FairShip takes next to no time, and MadGraph would be needed at runtime, so this should be in `requires`.